executor/api/rest/server.go

```diff
@@ -141,11 +141,13 @@ func (r *SeldonRestApi) Initialise() {
 			api01.Handle("/predictions", r.wrapMetrics(metric.PredictionHttpServiceName, r.predictions))
 			r.Router.NewRoute().Path("/api/v0.1/status/{" + ModelHttpPathVariable + "}").Methods("GET").HandlerFunc(r.wrapMetrics(metric.StatusHttpServiceName, r.status))
 			r.Router.NewRoute().Path("/api/v0.1/metadata/{" + ModelHttpPathVariable + "}").Methods("GET").HandlerFunc(r.wrapMetrics(metric.StatusHttpServiceName, r.metadata))
+			r.Router.NewRoute().PathPrefix("/api/v0.1/doc/").Handler(http.StripPrefix("/api/v0.1/doc/", http.FileServer(http.Dir("./openapi/v1.0/dist/"))))
```

Comment:
Is this the recommended approach for exposing API docs with OpenAPI? Actually, is there any recommended practice?

An alternative could be to just serve the `seldon.json` OpenAPI spec, and let people load that on their Swagger UIs. I'm not sure if and how that works though, but it could be less invasive than exposing the Swagger UI.